docker/stand-alone/.env

```diff
@@ -0,0 +1,13 @@
+# https://docs.docker.com/compose/env-file/
+# https://docs.docker.com/compose/environment-variables/#the-env-file
+
+# These values MUST be changed in order to be able to authenticate with SAM
+GOOGLE_OAUTH_CLIENT_ID=change_me
+GOOGLE_CLOUD_PROJECT_DOMAIN=change_me
```

Comment:
What is the `GOOGLE_CLOUD_PROJECT_DOMAIN`? Is it the project id in the GCP console? It doesn't seem like it, but I can't tell what it should be.

Reply:
I tried setting it to my the domain of my organization, ucsc.edu, and it seems to be working.